pkg/config/property.go

```diff
@@ -13,19 +13,19 @@
 package config
 
 const (
-	pluginRegistryURL = "plugin.registry.url"
+	pluginRegistryURL = "controller.plugin_registry.url"
 
 	routingSuffix        = "cluster.routing_suffix"
```

Comment:
that is used only for workspaceRoutings, right?
What do you think about
```
devworkspace.routing.default_class
devworkspace.routing.cluster_host_suffix
```
?

Reply:
I prefer a bit more `devworkspace.routing.cluster_host_suffix` than `cluster.routing_suffix` since on K8s infrastructure, we can assume that admin may use different hosts for different services(Che itself is deployed on one, workspaces on other), and such naming a bit clearly says that it's used for workspaces ingresses hosts suffix.
I'm OK with both options just choose whatever you find more logical.

P.S. On Che Server side it's called `che.infra.kubernetes.ingress.domain`.

Reply:
> devworkspace.routing.default_class

routingClass is in the [devfile spec](https://github.com/devfile/kubernetes-api/blob/28e3e6acdc368b503672738697b149dd23c86770/pkg/apis/workspaces/v1alpha1/devworkspace_types.go#L12) at this point, so it doesn't make sense to me to group it with controller-specific subresources.

Applied suggestion re: cluster_host_suffix, though -- you make a good point (and it was called `ingress.domain` here as well but we changed it since ingresses are more k8s-specific).